go/vt/sqlparser/parser.go

```diff
@@ -110,12 +110,14 @@ func Parse2(sql string) (Statement, BindVars, error) {
 func checkParserVersionFlag() {
 	if flag.Parsed() {
 		versionFlagSync.Do(func() {
-			convVersion, err := convertMySQLVersionToCommentVersion(*MySQLServerVersion)
-			if err != nil {
-				log.Error(err)
-				MySQLVersion = "50709" // default version if nothing else is stated
-			} else {
-				MySQLVersion = convVersion
+			MySQLVersion = "50709" // default version if nothing else is stated
```

Comment:
shouldn't we move this to the var declaration, so that before flags are parsed, we at least behave as 5.7?

Reply:
yes, that seems like a good idea. I'll do that